draft-wussler-openpgp-pqc.md

```diff
@@ -660,15 +660,17 @@ row "ECC-KEM" of {{tab-ecdh-cfrg-artifacts}}, {{tab-ecdh-nist-artifacts}}, and
 The operations `x25519Kem.encap()` and `x448Kem.encap()` are defined as
 follows:
 
- 1. Generate an ephemeral key pair {`v`, `V=vG`}, without any masking or
-    clamping as defined in [RFC7748]
+ 1. Generate an ephemeral key pair {`v`, `V=vG`} according to 
+    the procedure defined in RFC7748, that is, without any 
+    measures such as masking or clamping that are applied 
+    in previous OpenPGP specifications of this scheme.
```

Comment:
I think the part about previous specs is not necessary here, as it is already said in §1.3.1. The formulations here should be brief. I liked the old sentence better.